kotlin/src/main/kotlin/gildedrose/Item.kt

```diff
@@ -0,0 +1,7 @@
+package gildedrose
+
+open class Item(var name: String, var sellIn: Int, var quality: Int) {
```

Comment:
Data class? Not very idiomatic Kotlin, unless we're mutating its fields or the kata purpose is to refactor this.